website/docs.md

```diff
@@ -0,0 +1,42 @@
+# Introduction
+
+Cairo lint is a static code analysis tool for the Cairo language.
+
+It can help you improve your code quality and consistency by checking the codebase against a set of predefined rules, called lints. It can also automatically fix some of the issues found.
```

Comment:
```suggestion
It can help you improve your code quality and consistency by checking the codebase against a set of predefined rules, called lints.
It can also automatically fix some of the issues found.
```

put each sentence in separate line so that git diffs will be easier to grasp